src/install/ibm-mq/windows/install-windows.mdx

```diff
@@ -0,0 +1,40 @@
+---
+componentType: default
+headingText: Install the IBM MQ monitoring integration
+---
+
+1. Download the [nri-ibmmq MSI installer image](https://download.newrelic.com/infrastructure_agent/windows/integrations/nri-ibmmq/nri-ibmmq-amd64.msi).
+
+2. Run this command to install from the Windows command prompt:
+
+  ```shell
+  msiexec.exe /qn /i PATH\TO\nri-ibmmq-amd64.msi
+  ```
+
+3. Go to the integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`. Run this command to create a copy of the sample configuration file:
+
+  ```shell
+  copy ibmmq-config.yml.sample ibmmq-config.yml
+  ```
```

Comment:
Separate these into two steps, like: 

3. Navigate to the integrations directory:

```bash
cd \Program Files\New Relic\newrelic-infra\integrations.d
```

4. Create a copy of the sample configuration file:

```shell
copy ibmmq-config.yml.sample ibmmq-config.yml
```

```